### PR TITLE
Revamped exporter for wikidata

### PR DESCRIPTION
**Pull request by yusra-haider**
Revamped exporter for wikidata. The exporter now exports 5 json files named pathway.json, reaction.json, physicalEntity.json, modifiedProtein.json and parent.json for pathways, reactions, physical entities, modified proteins and child-parent links respectively

related PR: https://github.com/reactome/r-wikidata-bot/pull/2

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -106,7 +106,7 @@
                 <configuration>
                     <archive>
                         <manifest>
-                            <mainClass>org.reactome.server.tools.WikidataExport</mainClass>
+                            <mainClass>org.reactome.server.tools.Export</mainClass>
```
Comment (yusra-haider) [inline pom.xml:109]: @gsviteri should I update the version number here?

Comment (gsviteri) [reply pom.xml:109]: Where ?!

File: src/main/java/org/reactome/server/tools/Export.java [hunk 2]
```diff
@@ -0,0 +1,295 @@
...
+        if (physicalEntity instanceof EntityWithAccessionedSequence) {
+            List<AbstractModifiedResidue> mods = ((EntityWithAccessionedSequence) physicalEntity).getHasModifiedResidue();
+            if (mods != null && mods.size() > 0) {
+                return true;
+            }
+        }
+        return false;
+    }
+
+    private static void add_parent_child_link(String key, String val) {
```
Comment (gsviteri) [inline src/main/java/org/reactome/server/tools/Export.java:285]:
```suggestion
    private static void addParentChildLink(String key, String val) {
```

File: src/main/java/org/reactome/server/tools/WDLinks.java [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+
+import java.io.Serializable;
+import java.util.List;
+import java.util.Objects;
+
+/**
+ * @author Yusra Haider (yhaider@ebi.ac.uk)
+ **/
+
+// this class represents the children / parts for the different entities in Reactome
```
Comment (gsviteri) [inline src/main/java/org/reactome/server/tools/WDLinks.java:14]:
```suggestion
/**
 *  This class represents the children / parts for the different entities in Reactome
 **/
```

**Review by cqgong — APPROVED** [review]

File: pom.xml [hunk 4]
```diff
@@ -23,9 +23,9 @@
 
     <developers>
         <developer>
-            <id>skeating</id>
-            <name>Sarah Keating</name>
-            <email>skeating@ebi.ac.uk</email>
```
Comment (gsviteri) [inline pom.xml:28]: I would keep her name and Id...

**Review by gsviteri — APPROVED** [review]

**Pull request merged**